apps/masterbots.ai/lib/hooks/use-thread-visibility.tsx

```
@@ -0,0 +1,156 @@
+"use client"
+import React , { useEffect, useState } from 'react';
+import { getThreads, UpdateThreadVisibility, deleteThread, approveThread, getUnapprovedThreads } from '@/services/hasura';
+import { useSession } from 'next-auth/react';
+import { Thread } from 'mb-genql';
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)